hedgehog/src/Hedgehog/Gen.hs

```diff
@@ -1019,7 +1019,7 @@ uniqueInsert n xs kvs0 =
       [] ->
         Right xs
       (k, v) : kvs ->
-        uniqueInsert n (Map.insertWith (\x _ -> x) k v xs) kvs
+        uniqueInsert n (Map.insertWith const k v xs) kvs
```

Comment:
I don't think using `const` here is very obvious, the point of using a lambda above is you can see that we favour the left hand side